### PR TITLE
Add new test for util.py

### PR DESCRIPTION
**Pull request by nsiregar**
Add new test for `util.py` #14

### CODE REVIEW

File: readme.rst [hunk 1]
```diff
@@ -58,3 +58,10 @@ Requirements and dependencies
 - gidgethub
 - pyjwt
 - cryptography
+
+
+Running Tests
+=============================
```
Comment (Mariatta) [inline readme.rst]:
```suggestion
==========
```

The underline needs to be the same length as the text above it.

File: readme.rst [hunk 2]
```diff
@@ -58,3 +58,10 @@ Requirements and dependencies
...
+
+1. Install `pytest` and `pytest-tornasync`
```
Comment (Mariatta) [inline readme.rst]: I think it would be better to install dev-requirements.txt
```
python3 -m pip install -U -r dev-requirements.txt
```

Comment (Mariatta) [reply readme.rst]: Also, preferably in a virtual environment

File: readme.rst [hunk 3]
```diff
@@ -58,3 +58,10 @@ Requirements and dependencies
...
+2. Run `$ pytest tests/test_file.py`
```
Comment (Mariatta) [inline readme.rst]: `pytest tests`

**Comment by nsiregar** [comment]
Thanks, I've updated `readme.rst`.

**Comment by Mariatta** [comment]
Thanks! Sorry for such delay!

**Pull request merged**